### PR TITLE
RPi: Refresh default config.txt, add licence and warning to distroconfig.txt

### PR DESCRIPTION
**Pull request by MilhouseVH**
Prompted by comment from @popcornmix: https://github.com/LibreELEC/LibreELEC.tv/pull/389#issuecomment-225164630

I've removed the overclocking information but left the link - there are far better sources of information for this sort of thing than we can hope to cover in the config file. Also, the information in this file is inaccurate and will always remain inaccurate as the file is never updated on the users system.

### CODE REVIEW

File: packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt [hunk 1]
```diff
@@ -24,55 +22,26 @@
...
   gpu_mem_1024=256
 
 ################################################################################
-# Overclocking settings
-# WARNING: Do not change/enable if you do not know what you are doing!
-#          The System may become unstable or you can have data corruption or
-#          you can loose your warranty if you set wrong settings
-#
-# please read: http://elinux.org/RPi_config.txt#Overclocking_configuration
+# Various settings see also: http://elinux.org/RPi_config.txt
```
Comment (popcornmix) [inline packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt]: Can you link to https://www.raspberrypi.org/documentation/configuration/config-txt.md
That is official documentation and is more likely to be correct.

File: packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt [hunk 2]
```diff
@@ -24,55 +22,26 @@
...
-# try with this setting uncommented
-# If this still fails, try to use Medium mode without "over_voltage=2"
-# If it ultimately keeps failing, forget about this until it gets fixed
-# Use a more conservative mode that will works fine in any cases:
-# initial_turbo=30
-
-################################################################################
-# Overscan settings
-################################################################################
+# Set 'force_turbo=1' to disable dynamic overclocking.
```
Comment (popcornmix) [inline packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt]: Might want to make it explicit that this forces overclock settings always.
It sounds more like it is disabling overclocking.

File: packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt [hunk 3]
```diff
@@ -24,55 +22,26 @@
...
+force_turbo=0
 
 # Make display smaller to stop text spilling off the screen
-# see also http://www.raspberrypi.org/phpBB3/viewtopic.php?f=67&t=15700
-# overscan_scale=1
+# See also http://www.raspberrypi.org/phpBB3/viewtopic.php?f=67&t=15700
```
Comment (popcornmix) [inline packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt]: I've remove this link - not particularly useful.

File: packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt [hunk 4]
```diff
@@ -24,55 +22,26 @@
...
+# Note that the overscan settings only affect the splash screena and not Kodi.
 # disable_overscan=1
 
 # Adapt overscan values to your needs
```
Comment (popcornmix) [inline packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt]: I think I'd remove this. Considering the very limited benefit (a few of seconds of splash screen) I don't see many users trying to get this pixel perfect.
I'd just stick with the disable_overscan option which is probably good enough for most.

File: packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt [hunk 5]
```diff
@@ -24,55 +22,26 @@
...
-# Overscan settings
-################################################################################
+# Set 'force_turbo=1' to disable dynamic overclocking.
+force_turbo=0
 
 # Make display smaller to stop text spilling off the screen
-# see also http://www.raspberrypi.org/phpBB3/viewtopic.php?f=67&t=15700
-# overscan_scale=1
+# See also http://www.raspberrypi.org/phpBB3/viewtopic.php?f=67&t=15700
+# Note that the overscan settings only affect the splash screena and not Kodi.
```
Comment (popcornmix) [inline packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt]: "screena". You could mention the calibration menu is the right place to adjust kodi. You could also mention that TV settings are best place to solve this, but that may be going too far.

**Comment by MilhouseVH** [comment]
Thanks @popcornmix - updated.

File: packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt [hunk 6]
```diff
@@ -24,74 +22,39 @@
...
-# Overclocking settings
-# WARNING: Do not change/enable if you do not know what you are doing!
-#          The System may become unstable or you can have data corruption or
-#          you can loose your warranty if you set wrong settings
-#
-# please read: http://elinux.org/RPi_config.txt#Overclocking_configuration
+# Various settings see also: https://www.raspberrypi.org/documentation/configuration/config-txt.md
 ################################################################################
+# For overclocking information, please read:
+# http://elinux.org/RPi_config.txt#Overclocking_configuration
```
Comment (popcornmix) [inline packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt]: I prefer this link: https://www.raspberrypi.org/documentation/configuration/config-txt.md for overclocking info.
(unfortunately it doesn't support ``#Overclocking_configuration`` but perhaps you can merge this with previous line "Various settings including overclocking...")

File: packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt [hunk 7]
```diff
@@ -24,74 +22,39 @@
...
-# try with this setting uncommented
-# If this still fails, try to use Medium mode without "over_voltage=2"
-# If it ultimately keeps failing, forget about this until it gets fixed
-# Use a more conservative mode that will works fine in any cases:
-# initial_turbo=30
-
-################################################################################
-# Overscan settings
-################################################################################
+# Set 'force_turbo=1' to disable dynamic overclocking and enable overlcoking always.
```
Comment (popcornmix) [inline packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt]: "overlcoking"

**Comment by MilhouseVH** [comment]
Thanks - updated!

**Comment by popcornmix** [comment]
Looks good to me.

File: packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt [hunk 8]
```diff
@@ -1,21 +1,19 @@
 ################################################################################
-#      This file is part of OpenELEC - http://www.openelec.tv
-#      Copyright (C) 2009-2014 Stephan Raue (stephan@openelec.tv)
-#      Inspired and partly copied from:
-#  http://youresuchageek.blogspot.fr/2012/09/howto-raspberry-pi-openelec-on.html
+#      This file is part of LibreELEC - http://www.libreelec.tv
+#      Copyright (C) 2016 Team LibreELEC (team@libreelec.tv)
```
Comment (chewitt) [inline packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt]: That email address doesn't exist so best to leave it as "(c) 2016 LibreELEC"

File: packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/distroconfig.txt [hunk 9]
```diff
@@ -0,0 +1,19 @@
+################################################################################
+#      This file is part of LibreELEC - http://www.libreelec.tv
+#      Copyright (C) 2016 Team LibreELEC (team@libreelec.tv)
```
Comment (chewitt) [inline packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/distroconfig.txt]: same again :)

**Comment by MilhouseVH** [comment]
Updated (both).

**Pull request merged**